### PR TITLE
Add jscpd

### PR DESCRIPTION
**Pull request by jeeyo**

### CODE REVIEW

File: src/Provider/GitHub/GitHubPRService.ts [hunk 1]
```diff
@@ -31,6 +31,12 @@ export class GitHubPRService implements IGitHubPRService {
       userAgent: USER_AGENT,
       timeZone: TIME_ZONE,
       baseUrl: GitHubPRService.getApiBase(repoUrlObj),
+      log: {
+        debug: (message: string) => console.log('debug', message),
+        error: (message: string) => console.log('error', message),
+        info: (message: string) => console.log('info', message),
+        warn: (message: string) => console.log('warn', message),
+      },
```
Comment (encX) [inline src/Provider/GitHub/GitHubPRService.ts:39]: Use winston for logging (with correct log levels) like the rest of the app please

**Review by encX — COMMENTED** [review]
Check comment please

File: src/Logger/Logger.ts [hunk 2]
```diff
@@ -3,7 +3,7 @@ import { createLogger, format, transports } from 'winston';
 const { combine, prettyPrint, timestamp, simple } = format;
 
 export const Log = createLogger({
-  silent: process.env['NODE_ENV'] === 'test', // set by jest
+  // silent: process.env['NODE_ENV'] === 'test', // set by jest
```
[the remark this answers is not in the document]
Comment (jeeyo) [reply src/Logger/Logger.ts:6]: this is for debugging, I accidentally committed it
will remove it

File: src/Provider/GitHub/GitHubPRService.ts (hunk 1 above)
Comment (jeeyo) [reply src/Provider/GitHub/GitHubPRService.ts:39]: this is also for debugging
will remove it

**Review by encX — APPROVED** [review]

**Pull request merged**